### ISSUE TITLE
Handheld search icon refreshes page

### ISSUE THREAD

**Issue by jrick1229** [woocommerce/storefront#1801]
## Describe the bug

When accessing a site using Storefront on mobile/tablet, clicking the 'Search' icon in the handheld menu refreshes the page instead of opening the search box: https://d.pr/i/4Rp7GV

Ticket: 4329816-zen

Isolating the problem (mark completed items with an [x]):
- [x] I have deactivated other plugins and themes and confirmed this bug occurs when only WooCommerce + Storefront theme are active.
- [x] I can reproduce this bug consistently using the steps below.

## To Reproduce

1. Activate Storefront
2. access site from a handheld device
3. tap the search icon
4. page will reload

## Screenshots

Recording in Safari on iPhone XR: 

https://user-images.githubusercontent.com/19918755/134960929-e270471c-7be8-4f3c-82ea-356151b6f84d.MP4

## Expected behavior

Tapping the 'Search' icon should bring up a search box.

### Browser Environment

- Platform: iPhone XR (v14.8)
- Browser(s): Safari

### WordPress Environment
Please provide relevant details of your WordPress setup and server environment.

<details>

```
### WordPress Environment ###

WordPress address (URL): http://testsite10.local
Site address (URL): http://testsite10.local
WC Version: 5.7.1
REST API Version: ✔ 5.7.1
WC Blocks Version: ✔ 5.7.2
Action Scheduler Version: ✔ 3.2.1
WC Admin Version: ✔ 2.6.5
Log Directory Writable: ✔
WP Version: 5.8.1
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: ✔
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: nginx/1.16.0
PHP Version: 7.3.5
PHP Post Max Size: 1,000 MB
PHP Time Limit: 1200
PHP Max Input Vars: 4000
cURL Version: 7.54.0
LibreSSL/2.6.5

SUHOSIN Installed: –
MySQL Version: 8.0.16
Max Upload Size: 300 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 5.7.1
WC Database Prefix: wp_
Total Database Size: 154.56MB
Database Data Size: 133.16MB
Database Index Size: 21.40MB
wp_woocommerce_sessions: Data: 0.16MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.08MB + Index: 0.05MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 1.52MB + Index: 0.56MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.22MB + Index: 0.22MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 1.31MB + Index: 0.36MB + Engine InnoDB
wp_advanced_notification_triggers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_advanced_notifications: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_automatewoo_abandoned_carts: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_automatewoo_customer_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_automatewoo_customers: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_automatewoo_events: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_automatewoo_guest_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_automatewoo_guests: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_automatewoo_log_meta: Data: 0.20MB + Index: 0.09MB + Engine InnoDB
wp_automatewoo_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_automatewoo_queue: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_automatewoo_queue_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_automatewoo_referral_advocate_keys: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_automatewoo_referral_invites: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_automatewoo_referrals: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_ce4wp_abandoned_checkout: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 1.52MB + Index: 0.78MB + Engine InnoDB
wp_e_submissions: Data: 0.02MB + Index: 0.23MB + Engine InnoDB
wp_e_submissions_actions_log: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_e_submissions_values: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_followup_coupon_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_coupons: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_customer_carts: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_customer_notes: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_customer_orders: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_followup_customers: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_followup_email_excludes: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_email_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_email_orders: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_followup_email_tracking: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_followup_followup_history: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_order_categories: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_order_items: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_followup_subscriber_lists: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_subscribers: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_followup_subscribers_to_lists: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_fusion_form_entries: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_fusion_form_fields: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_fusion_form_submissions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_fusion_forms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_draft_submissions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_gf_entry: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_entry_meta: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_gf_entry_notes: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_form: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gf_form_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_gf_form_revisions: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_gf_form_view: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_mollie_pending_payment: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_ms_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_options: Data: 86.03MB + Index: 0.17MB + Engine InnoDB
wp_postmeta: Data: 10.52MB + Index: 13.97MB + Engine InnoDB
wp_posts: Data: 1.48MB + Index: 0.70MB + Engine InnoDB
wp_say_what_strings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_term_relationships: Data: 0.28MB + Index: 0.19MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 1.06MB + Index: 0.06MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_wc_booking_relationships: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_bookings_availability: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_bookings_availabilitymeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_deposits_payment_plans: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_deposits_payment_plans_schedule: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_mewz_wcas_match_rows: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_mewz_wcas_match_sets: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.09MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.09MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_points_rewards_user_points: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_points_rewards_user_points_log: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.28MB + Index: 0.63MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wcpv_commissions: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wcpv_per_product_shipping_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wcrs_logs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wcs_payment_retries: Data: 0.11MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_bundled_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_bundled_items: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_exported_csv_items: Data: 1.52MB + Index: 0.13MB + Engine InnoDB
wp_woocommerce_gc_activity: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_gc_cards: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_per_product_shipping_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_pickup_locations_geodata: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_woocommerce_shipping_flat_rate_boxes: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_quotes: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_shipping_table_rates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_square_customers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_subscription_downloads: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpml_mails: Data: 24.52MB + Index: 0.00MB + Engine InnoDB
wp_yoast_indexable: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_yoast_indexable_hierarchy: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_primary_term: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_yoast_seo_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 21
aw_workflow: 15
bookable_person: 2
bookable_resource: 2
cbk_cashback: 1
cbk_cashback_log: 3
cbk_master_log: 3
cbk_order_rules: 1
cpn_rm_customer: 13
custom_css: 1
custom-css-js: 2
customize_changeset: 3
def_reg_fields: 10
elementor_library: 1
fgf_rules: 1
follow_up_email: 1
global_product_addon: 1
nav_menu_item: 4
oembed_cache: 7
page: 35
post: 2
product: 1820
product_variation: 878
qts_form_field: 12
revision: 83
shipping_package: 3
shop_bogof_rule: 1
shop_coupon: 37
shop_order: 578
shop_order_refund: 9
shop_subscription: 110
wc_booking: 9
wc_membership_plan: 2
wc_pickup_location: 3
wc_user_membership: 1
wcpf_item: 4
wcpf_project: 1
wishlist: 1
ww_stats: 1

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ❌ Error messages should not be shown to visitors.

### Active Plugins (1) ###

WooCommerce: by Automattic – 5.7.1

### Inactive Plugins (30) ###

Added to Cart Popup: by KoalaApps – 1.0.0
AutomateWoo: by WooCommerce – 5.5.3
AutomateWoo - Refer A Friend Add-on: by WooCommerce – 2.6.2
Custom Product Boxes: by CodeIncept – 1.0.5
Gr8pay Payment Gateway: by OnePix – 1.1.5
Paywall for WooCommerce: by TIV.NET INC – 3.3.0
Product Filters for WooCommerce: by WooCommerce – 1.2.5
Say What?: by Ademti Software – 2.0.2
WooCommerce added to cart popup (Ajax): by XootiX – 1.5
WooCommerce All Products For Subscriptions: by SomewhereWarm – 3.1.29 (update to version 3.1.30 is available)
WooCommerce Blocks: by Automattic – 5.9.1
WooCommerce Bookings: by WooCommerce – 1.15.43
WooCommerce Checkout Field Editor: by WooCommerce – 1.6.2 (update to version 1.7.0 is available)
WooCommerce Customizer: by SkyVerge – 2.7.4
WooCommerce Dynamic Pricing: by Element Stark – 3.1.26
WooCommerce FedEx Shipping: by WooCommerce – 3.4.36 (update to version 3.4.37 is available)
WooCommerce Local Pickup Plus: by SkyVerge – 2.9.8
WooCommerce Memberships: by SkyVerge – 1.22.4 (update to version 1.22.5 is available)
WooCommerce Payments: by Automattic – 2.9.1 (update to version 3.0.0 is available)
WooCommerce PayPal Payments: by WooCommerce – 1.5.1
WooCommerce Points and Rewards: by WooCommerce – 1.7.1
WooCommerce Points and Rewards Fix Individual Coupons: by Maciej Krawczyk – 1.0.0
WooCommerce Product Add-ons: by WooCommerce – 4.1.0 (update to version 4.2.0 is available)
WooCommerce Ship to Multiple Addresses: by WooCommerce – 3.6.33
WooCommerce Stripe fee in Report: by studio RVOLA – 1.2.6
WooCommerce Stripe Gateway: by WooCommerce – 5.5.0
WooCommerce Subscriptions: by WooCommerce – 3.1.4
WooCommerce Table Rate Shipping: by WooCommerce – 3.0.33
WP Crontrol: by John Blackbourn & crontributors – 1.10.0
WP Mail Logging: by Wysija – 1.9.9

### Must Use Plugins (1) ###

Local WP Live Link Helper: by Flywheel – 2.0

### Settings ###

API Enabled: –
Force SSL: ✔
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: accommodation-booking (accommodation-booking)
booking (booking)
bundle (bundle)
cpb_custom_product_boxes (cpb_custom_product_boxes)
external (external)
grouped (grouped)
paywall (paywall)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude_from_catalog (exclude_from_catalog)
exclude_from_search (exclude_from_search)
exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: ✔

### WC Pages ###

Shop base: #6 - /shop/
Cart: #7 - /cart/
Checkout: #403 - /checkout-block/
My account: #9 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 3.9.0
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Complete: 548
Oldest: 2021-08-30 10:36:30 -0400
Newest: 2021-09-27 14:00:58 -0400

Pending: 29
Oldest: 2021-09-28 08:30:56 -0400
Newest: 2023-06-30 09:24:28 -0400

### Status report information ###

Generated at: 2021-09-27 14:03:00 -04:00
```
</details>

**Comment by tjcafferkey** [comment]
@jrick1229 I'm unable to reproduce this issue. Please could you try updating Storefront to the latest version and trying again? 

There was a known issue with version 3.9.0 (if you're running that) but perhaps this was fixed in version 3.9.1 which is currently the latest.

**Comment by jrick1229** [comment]
Hi @tjcafferkey – updating to v3.9.1 seems to have fixed this on my local site! 😁 Thank you
